process-compose-mp.yml

```diff
@@ -0,0 +1,117 @@
+version: "3"
+
+environment:
+  - ESPRESSO_SEQUENCER_GENESIS_FILE=data/genesis/demo-marketplace.toml
+  - ESPRESSO_BUILDER_GENESIS_FILE=data/genesis/demo-marketplace.toml
+processes:
+
+  marketplace-reserve-builder:
+    command: marketplace-builder
+    ports:
+      - "$ESPRESSO_RESERVE_BUILDER_SERVER_PORT:$ESPRESSO_RESERVE_BUILDER_SERVER_PORT"
+    environment:
+      - ESPRESSO_BUILDER_SERVER_PORT=$ESPRESSO_RESERVE_BUILDER_SERVER_PORT
+      - ESPRESSO_MARKETPLACE_BUILDER_IS_RESERVE=true
+      - ESPRESSO_MARKETPLACE_SOLVER_API_URL=http://localhost:$ESPRESSO_MARKETPLACE_SOLVER_API_PORT
+      - ESPRESSO_MARKETPLACE_BUILDER_NAMESPACE=10000
+      - ESPRESSO_SEQUENCER_STATE_PEERS=http://localhost:$ESPRESSO_SEQUENCER_API_PORT
+      - ESPRESSO_SEQUENCER_HOTSHOT_EVENT_STREAMING_API_URL=http://localhost:$ESPRESSO_SEQUENCER_HOTSHOT_EVENT_STREAMING_API_PORT
+      - ESPRESSO_BUILDER_GENESIS_FILE
+    depends_on:
+      sequencer0:
+        condition: process_healthy
+      fund-builder:
+        condition: process_completed
+    readiness_probe:
+      http_get:
+        scheme: http
+        host: localhost
+        port: $ESPRESSO_RESERVE_BUILDER_SERVER_PORT
+        path: /healthcheck
+      failure_threshold: 100
+
+  marketplace-fallback-builder:
+    command: marketplace-builder
+    ports:
+      - "$ESPRESSO_FALLBACK_BUILDER_SERVER_PORT:$ESPRESSO_FALLBACK_BUILDER_SERVER_PORT"
+    environment:
+      - ESPRESSO_BUILDER_SERVER_PORT=$ESPRESSO_FALLBACK_BUILDER_SERVER_PORT
+      - ESPRESSO_MARKETPLACE_BUILDER_IS_RESERVE=false
+      - ESPRESSO_MARKETPLACE_SOLVER_API_URL=http://localhost:$ESPRESSO_MARKETPLACE_SOLVER_API_PORT
+      - ESPRESSO_MARKETPLACE_BUILDER_NAMESPACE=10000
+      - ESPRESSO_SEQUENCER_STATE_PEERS=http://localhost:$ESPRESSO_SEQUENCER_API_PORT
+      - ESPRESSO_SEQUENCER_HOTSHOT_EVENT_STREAMING_API_URL=http://localhost:$ESPRESSO_SEQUENCER_HOTSHOT_EVENT_STREAMING_API_PORT
+      - ESPRESSO_BUILDER_GENESIS_FILE
+    depends_on:
+      sequencer0:
+        condition: process_healthy
+      fund-builder:
+        condition: process_completed
+    readiness_probe:
+      http_get:
+        scheme: http
+        host: localhost
+        port: $ESPRESSO_FALLBACK_BUILDER_SERVER_PORT
+        path: /healthcheck
+      failure_threshold: 100
+
+  permissionless-builder:
+    disabled: true
+
+  submit-transactions-public:
+    command: submit-transactions
+    environment:
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MIN_NAMESPACE=1
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MAX_NAMESPACE=6
+
+  submit-transactions-private:
+    disabled: true
```

Comment:
@tbro Not sure if that's the only issue but if we don't run anything on the `ESPRESSO_SUBMIT_TRANSACTIONS_PRIVATE_PORT` we need to check the health check

https://github.com/EspressoSystems/espresso-sequencer/blob/f5e7be1e49e3492e6c462f367878b07a9670d645/scripts/smoke-test-demo#L17

https://github.com/EspressoSystems/espresso-sequencer/blob/f5e7be1e49e3492e6c462f367878b07a9670d645/scripts/smoke-test-demo#L72